fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -179,34 +168,30 @@
     private static final Set<Property> ldpManagedProperties = of(CONTAINS);
 
     // REPOSITORY INFORMATION
-    public static final Property HAS_OBJECT_COUNT =
+    private static final Property HAS_OBJECT_COUNT =
```

Comment:
`HAS_OBJECT_COUNT` and `HAS_OBJECT_SIZE` are never populated since having removed the metrics code, so these should be removeable (I think they are already blocked from being added via user requests because of the namespace)